libtelemetry/src/main/java/com/mapbox/android/telemetry/ConcurrentQueue.java

```diff
@@ -8,35 +8,32 @@
 
 class ConcurrentQueue<T> {
```

Comment:
IMHO this class should be put to bed completely, i don't see a reason in maintaining additional generic wrapper for `ConcurrentLinkedQueue`